nitime/timeseries.py

```diff
@@ -985,43 +981,43 @@ def __repr__(self):
     # add some methods that implement arithmetic on the timeseries data
     def __add__(self, other):
         out = self.copy()
-        out.data = out.data.__add__(other)
+        out.data = out.data.__add__(np.array(other).T)
```

Comment:
Good call on `asanyarray`. The transpose surprised me -- I don't think we ever tested addition with n-dimensional arrays before, though. I can't really reason about why this would be necessary, but y'know, proof by unit-test.